lightblue-ldap-config/src/main/java/com/redhat/lightblue/config/ldap/LdapDBResolver.java

```diff
@@ -71,20 +70,20 @@ private LdapDataSourceConfiguration findByDatabase(String database){
         return null;
     }
 
-	@Override
-	public List<LDAPConnection> getConnections() {
-		
-		List<LDAPConnection> ldapConnections = new ArrayList<>();
-		
-		for (LdapDataSourceConfiguration ldapDS : ldapDataSources){
-			try {
-				ldapConnections.add(ldapDS.getLdapConnection());
-			} catch (LDAPException e) {
-				ldapConnections.add(null);
-			}
-		}
-		
-		return Collections.unmodifiableList(ldapConnections);
-	}
+    @Override
+    public Map<String, Object> getLDAPConnectionsStatus() {
+        
+        Map<String, Object> connectionsStatus = new HashMap<>();
+        LDAPConnection connection = null;
+        for (LdapDataSourceConfiguration ldapDS : ldapDataSources) {
+            try {
+                connection = ldapDS.getLdapConnection();
+                connectionsStatus.put(ldapDS.getDatabaseName(), connection.isConnected());
```

Comment:
Is it possible to have multiple configurations that have the same database name? If that happened, the map would hide connections with duplicate db names.